File: 3227-find-missing-and-repeated-values/find-missing-and-repeated-values.py
class Solution(object):
    def findMissingAndRepeatedValues(self, grid):
        """
        :type grid: List[List[int]]
        :rtype: List[int]
        """
        n=len(grid)
        ans=[0]*2
        track=[0]*(n*n)
        for i in range(n):
            for j in range(n):
                val=grid[i][j]
                track[val-1]+=1
        a=0
        b=0
        for i in range(n*n):
            if track[i]==2:
                a=i+1
            elif track[i]==0:
                b=i+1
        ans[0]=a
        ans[1]=b
                 
        return ans
    

File: 3227-find-missing-and-repeated-values/find-missing-and-repeated-values.cpp
class Solution {
public:
    vector<int> findMissingAndRepeatedValues(vector<vector<int>>& grid) {
        int n=grid.size();
        vector<int> ans;
        vector<int> track(n*n, 0);
        for(int i=0;i<n;i++)
        {
            for(int j=0;j<n;j++)
            {
                int val=grid[i][j];
                track[val-1]+=1;
            }
        }
        int a=0,b=0;
        for(int i=0;i<n*n;i++)
        {
            if(track[i]==2)
            {
                a=i+1;
            }
            else if(track[i]==0)
            {
               b=i+1;
            }
        }
        ans.push_back(a);
        ans.push_back(b);
                 
        return ans;
    }
};